pandas/tests/frame/test_constructors.py

```diff
@@ -2466,7 +2468,7 @@ def test_constructor_list_str_na(self, string_dtype):
         tm.assert_frame_equal(result, expected)
 
     @pytest.mark.parametrize("copy", [False, True])
-    @td.skip_array_manager_not_yet_implemented
+    @td.skip_array_manager_invalid_test  # TODO(ArrayManager)
```

Comment:
why invalid?

Reply:
Yeah, not ideal, but I did that because that still skips instead of xfails .. And properly xfailing would give a quite complex selection of which cases already pass and which not (not all parametrization combinations fail with ArrayManager). But added to TODO marker to indicate this is still something to fix, and not a test to ignore.

Reply:
The condition would be:

```
        if using_array_manager and not (
            (any_numpy_dtype in (tm.STRING_DTYPES + tm.BYTES_DTYPES))
            or (
                any_numpy_dtype
                in (tm.DATETIME64_DTYPES + tm.TIMEDELTA64_DTYPES + tm.BOOL_DTYPES)
                and copy
            )
        ):
            td.mark_array_manager_not_yet_implemented(request)
```

Can also add it if you prefer, either way (don't think the condition on itself is very useful / adds noise, but being able to use xfail is certainly useful)

Reply:
since you've already written it out id say might as well use it.  correct-but-not-pretty beats the other way around

Reply:
thanks